README.md

```diff
@@ -511,7 +532,25 @@ block-beta
 ```bash
 protoflow execute WriteFile path=/tmp/file.txt
 ```
+#### [`WriteSocket`]
```

Comment:
Leave an empty line prior to the section header

Reply:
done